ods_ci/tests/Resources/Files/pipeline-samples/v2/iris_pipeline.py

```diff
@@ -66,57 +53,48 @@ def train_model(
     import pickle
 
     import pandas as pd
-    from sklearn.model_selection import train_test_split
+    from sklearn.metrics import confusion_matrix, roc_curve
+    from sklearn.model_selection import cross_val_predict, train_test_split
     from sklearn.neighbors import KNeighborsClassifier
 
-    from sklearn.metrics import roc_curve
-    from sklearn.model_selection import train_test_split, cross_val_predict
-    from sklearn.metrics import confusion_matrix
-
-
     with open(normalized_iris_dataset.path) as f:
         df = pd.read_csv(f)
 
-    y = df.pop('Labels')
+    y = df.pop("Labels")
     X = df
 
     X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
 
     clf = KNeighborsClassifier(n_neighbors=n_neighbors)
     clf.fit(X_train, y_train)
 
-    predictions = cross_val_predict(
-        clf, X_train, y_train, cv=3)
+    predictions = cross_val_predict(clf, X_train, y_train, cv=3)
     metrics.log_confusion_matrix(
-        ['Iris-Setosa', 'Iris-Versicolour', 'Iris-Virginica'],
-        confusion_matrix(
-            y_train,
-            predictions).tolist()  # .tolist() to convert np array to list.
+        ["Iris-Setosa", "Iris-Versicolour", "Iris-Virginica"],
+        confusion_matrix(y_train, predictions).tolist(),  # .tolist() to convert np array to list.
     )
 
-    model.metadata['framework'] = 'scikit-learn'
-    with open(model.path, 'wb') as f:
+    model.metadata["framework"] = "scikit-learn"
+    with open(model.path, "wb") as f:
         pickle.dump(clf, f)
 
 
-@dsl.pipeline(name='iris-training-pipeline')
+@dsl.pipeline(name="iris-training-pipeline")
 def my_pipeline(
     standard_scaler: bool = True,
     neighbors: int = 3,
 ):
     create_dataset_task = create_dataset()
 
     normalize_dataset_task = normalize_dataset(
-        input_iris_dataset=create_dataset_task.outputs['iris_dataset'],
-        standard_scaler=True)
+        input_iris_dataset=create_dataset_task.outputs["iris_dataset"], standard_scaler=True
+    )
 
     train_model(
-        normalized_iris_dataset=normalize_dataset_task
-        .outputs['normalized_iris_dataset'],
-        n_neighbors=neighbors)
+        normalized_iris_dataset=normalize_dataset_task.outputs["normalized_iris_dataset"], n_neighbors=neighbors
+    )
+
 
-endpoint = 'http://ml-pipeline-ui-kubeflow.apps.rmartine.dev.datahub.redhat.com/'
+endpoint = "http://ml-pipeline-ui-kubeflow.apps.rmartine.dev.datahub.redhat.com/"
```

Comment:
> ```endpoint = "http://ml-pipeline-ui-kubeflow.apps.rmartine.dev.datahub.redhat.com/"```
Using http protocol is insecure. Use https instead

Reply:
I'd ignore this

Reply:
I don't mind if you put it to ignore, but this bit seems to me like something that is irrelevant or will be overwritten on the actual run? I don't think this hostname will be working.

Reply:
This sample pipeline is a verbatim copy from https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/docs/example_pipelines/iris/iris-pipeline.py. 

Currently in ods-ci we use the compiled version (iris_pipeline.yaml), which doesn't use that endpoint

Reply:
That's helpful, so probably the correct formatter fix should've been to disable formatter for this file, actually